Makefile

```
@@ -206,8 +209,11 @@ install-service-systemd: service-systemd
 	install -Dm644 dunst.systemd.service ${DESTDIR}${SERVICEDIR_SYSTEMD}/dunst.service
 endif
 
+ifeq (1,${LIBNOTIFY})
```

I think this should be
```suggestion
ifneq (0,${LIBNOTIFY})
```
Otherwise this results in false when no variable is defined. It should default to true, as stated in the README.

---

I was on a system without libnotify, so this wasn't tested. thank you!